### PR TITLE
Bulk scan phase 2-Private Law non Edge Cases-Child Arrangement- Part …

### PR DESCRIPTION
**Pull request by RonanCog**
…15&16

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/resources/application-c100-transform.yaml [hunk 1]
```diff
@@ -164,3 +164,16 @@ uk:
...
+              checklistSection7Completed: section7RelatingToOtherCourtFullyCompleted
+              checklistSection8To10Completed: section8Section9Section10FactorsAffectingProceedingsCompleted
+              checklistSection11Completed: section11RelatingToApplicantCompleted
+              checklistSection12To14Completed: sections12Section13Section14RelatingToRespondentAndOthersNoticeCompleted
+              checklistSection16Completed: section16RelatingToTruthStatementCompleted
+              printFullName: printFullName
+              agreedBy: applicantSolicitorsFirmName
+              SignedDate: applicantAndApplicantsolicitorSignedDate
+              positionOrOfficeHeld: position_or_OfficeHeld
+              courtFee: courtFee
```
Comment (hmcts-kodatala) [inline src/main/resources/application-c100-transform.yaml:179]: can you change  to courtFeeConsent instead of courtFee?

**Review by hmcts-kodatala — COMMENTED** [review]
pls correct all review comments

**Review by hmcts-kodatala — APPROVED** [review]
OK

**Review by Murshadhaque — APPROVED** [review]

File: src/main/resources/application-c100-transform.yaml (hunk 1 above)
Comment (RonanCog) [reply src/main/resources/application-c100-transform.yaml]: This has been changed

**Pull request merged**